extensions/wikia/AdEngine/js/video/uiTemplate.js

```diff
@@ -21,11 +21,34 @@ define('ext.wikia.adEngine.video.player.uiTemplate', [], function () {
 			'volumeControl',
 			'toggleVideo',
 			'replayOverlay'
+		],
+		clickToPlaySplitLayout = [
+			'progressBar',
+			'pauseOverlay',
+			'volumeControl',
+			'toggleVideo',
+			'replayOverlay',
+			'closeButton'
 		];
 
+	function selectTemplate(videoSettings) {
+		var template = defaultLayout;
+
+		if (!videoSettings.isAutoPlay() && videoSettings.isSplitLayout()) {
+			template = clickToPlaySplitLayout;
+		} else if (videoSettings.isSplitLayout()) {
+			template = splitLayout;
+		} else if (videoSettings.isAutoPlay()) {
+			template = autoPlayLayout;
+		}
+
```

Comment:
Maybe it would be useful to leave this debug message?
`log(['VUAP UI elements', template], log.levels.debug, logGroup);`